addons/mail/static/tests/messaging_menu/messaging_menu_tests.js

```diff
@@ -674,33 +682,27 @@ QUnit.test("<br/> tags in message body preview are transformed in spaces", async
     await contains(".o-mail-NotificationItem-text", { text: "You: a b c d" });
 });
 
-QUnit.test(
-    "Messaging menu notification body of chat should show author name once",
-    async (assert) => {
-        const pyEnv = await startServer();
-        const partnerId = pyEnv["res.partner"].create({ name: "Demo User" });
-        const channelId = pyEnv["discuss.channel"].create({
-            channel_type: "chat",
-            channel_member_ids: [
-                Command.create({ partner_id: pyEnv.currentPartnerId }),
-                Command.create({ partner_id: partnerId }),
-            ],
-        });
-        pyEnv["mail.message"].create({
-            author_id: partnerId,
-            body: "<p>Hey!</p>",
-            model: "discuss.channel",
-            res_id: channelId,
-        });
-        await start();
-        await click(".o_menu_systray i[aria-label='Messages']");
-        assert.strictEqual(
-            (await contains(".o-mail-NotificationItem"))[0].textContent.split("Demo User").length -
-                1,
-            1
-        );
-    }
-);
```

Comment:
Assertions look simpler... but it leaks part names `-name` and `-text`.
At least the previous test was looking in whole text content of the notification item to see it appears only once.
With the new diff, the test may pass although it's incorrect (some other parts may have "Demo User"...
...So the test can be a lie